v2/contribute/checklists/saas/mysql-rds.md

```diff
@@ -1,3 +1,8 @@
+---
+id: mysql-rds
+title: Launching new Mysql RDS Cluster
+hide_title: true
+---
```

Comment:
- How come this is added in this PR but the file existed before? Was this forgotten to be added before?
- Also, describe why we are using a MySQL RDS when we already have a postgresql rds section in the new-region.md page.

Reply:
> How come this is added in this PR but the file existed before? Was this forgotten to be added before?

Yes, it was forgotten to be added

> Also, describe why we are using a MySQL RDS when we already have a postgresql rds section in the new-region.md page.

There was mysql connection pool error where the old connections were not removed because the mysql procedure was not set. I made that change in DB and you asked me to put the changes in this file for reference so that if we ever move/create new mysql rds instance, that issue doesn't occur